tests/TaskLineRenderer.test.ts

```diff
@@ -596,3 +598,60 @@ describe('task line rendering', () => {
         );
     });
 });
+
+describe('Visualise HTML', () => {
+    async function renderAndVerifyHTML(task: Task, layoutOptions: LayoutOptions) {
+        const parentRender = await createMockParentAndRender(task, layoutOptions);
+        const taskAsHTML = parentRender.innerHTML
+            .replace(/ data-/g, '\n    data-')
+            .replace(/<span/g, '\n        <span');
+
+        verifyWithFileExtension(taskAsHTML, 'html');
+    }
+
+    const fullTask = TaskBuilder.createFullyPopulatedTask();
+    const minimalTask = fromLine({ line: '- [-] empty' });
+
+    function layoutOptionsFullMode(): LayoutOptions {
+        const layoutOptions = new LayoutOptions();
+
+        // Show every Task field (not entirely true)
```

Comment:
I suggested that `not entirely true` comment when we were writing throw-away code...
Now it's staying, it's fairly unhelpful - how would anyone know from the comment what bits are untrue?

Reply:
indeed, tried to be more explicit now